packages/react/package.json

```
@@ -78,6 +78,6 @@
     "vite-plugin-dts": "^3.9.1",
     "vite-plugin-linter": "^2.1.1",
     "vite-tsconfig-paths": "^4.3.2",
-    "yjs": "^13.6.19"
+    "yjs": "^13.6.20"
```

It should be a `peerDependency`

```suggestion
    "yjs": "^13.6.20"
```